app/reschedule/route.js

```diff
@@ -65,25 +70,26 @@ export default Route.extend(StaffRouteMixin, ReportFilterRouteMixin, {
      * @method verifyPage
      * @public
      */
-    async verifyPage() {
-      try {
-        this.send('loading')
+      async verifyPage() {
+        try {
+          this.send('loading')
 
-        let params = this.get('params')
+          let params = this.get('params')
 
-        let queryString = toQueryString(cleanParams(params))
+          let queryString = toQueryString(cleanParams(params))
 
-        let url = `/api/v1/reports/verify?${queryString}`
+          let url = `/api/v1/reports/verify?${queryString}`
 
-        await this.get('ajax').request(url, { method: 'POST' })
+          await this.get('ajax').request(url, { method: 'POST' })
 
-        this.refresh()
-      } catch (e) {
-        /* istanbul ignore next */
-        this.get('notify').error('Error while verifying the page')
-      } finally {
-        this.send('finished')
+          this.refresh()
+        } catch (e) {
+          /* istanbul ignore next */
+          this.get('notify').error('Error while verifying the page')
+        } finally {
+          this.send('finished')
+        }
       }
     }
   }
-})
+)
```

Comment:
Prettier updates .... ^^

Reply:
Ah, not even. I thought this was because of the new way prettier is laying out chained function calls. nvmd

Reply:
Wait a second.. That's weird..

Reply:
OHGODNO that's ugly..

Reply:
```javascript
const BaseRoute = Route.extend(
  StaffRouteMixin,
  ReportFilterRouteMixin,
  DesktopOnlyRouteMixin,
  {}
)

/**
 * Route for filtering and rescheduling reports
 *
 * @class RescheduleRoute
 * @extends Ember.Route
 * @using StaffRouteMixin
 * @using ReportFilterRouteMixin
 * @using DesktopOnlyRouteMixin
 * @public
 */
export default BaseRoute.extend({ ... })
```
Worse or better?

Reply:
We're using prettier - there is no more "ugly" or "beautiful" just "prettier-conformant" or not :stuck_out_tongue_closed_eyes:

Reply:
better!